src/main/java/io/github/syst3ms/skriptparser/sections/SecSwitch.java

```diff
@@ -0,0 +1,116 @@
+package io.github.syst3ms.skriptparser.sections;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.effects.EffCase;
+import io.github.syst3ms.skriptparser.lang.*;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Basic switch control statement. Only {@link SecCase case} sections/effects are allowed within this section.
+ * The given expression will be matched against each case and all matching ones will be run.
+ * Note that unlike in Java, each case is run separately.
+ * <br>
+ * <pre>
+ *     switch (2) {
+ *         case 1:
+ *             print("Hello");
+ *         case 2:
+ *             print("World!");
+ *             break;
+ *         default:
+ *             print("Nothing");
+ *     }
+ * </pre>
+ * Would result in the following Skript code:
+ * <br>
+ * <pre>
+ *     switch 2:
+ *         case 1:
+ *             print "Hello"
+ *         case 1 or 2:
+ *             print "World!"
+ *         default:
+ *             print "Nothing"
+ * </pre>
+ * Note that the default part is only executed if no match was found.
+ *
+ * @name Switch
+ * @type SECTION
+ * @pattern (switch|for) %object%
+ * @since ALPHA
+ * @author Mwexim
+ * @see SecCase
+ * @see EffCase
+ */
+@SuppressWarnings("unchecked")
+public class SecSwitch extends CodeSection {
+    static {
+        Parser.getMainRegistration().addSection(
+                SecSwitch.class,
+                "(switch|for) %object%"
```

Comment:
Hmmm, following the discussion, it would be nice to find a more englishy pattern that would go along with "when" ("for" is bound to confuse people btw).

Reply:
Instead of "when" for the section syntax, we could use ``with`` as
```python
with {_role}:
	when "MEMBER":
		send "Welcome back!"
	when "ADMIN":
		send "Nice to see you again admin!"
	otherwise:
		send "Welcome new player!"
```
Plus, if {_role} is empty, the section should run anyway, but execute directly the ``otherwise`` section

Reply:
> Instead of "when" for the section syntax, we could use `with` as
> 
> ```python
> with {_role}:
> 	when "MEMBER":
> 		send "Welcome back!"
> 	when "ADMIN":
> 		send "Nice to see you again admin!"
> 	otherwise:
> 		send "Welcome new player!"
> ```
Hum imo it's confused, because with is frequently used to speak of a thing used for (idk if i'm clear)
> Plus, if {_role} is empty, the section should run anyway, but execute directly the `otherwise` section
I agree

Reply:
``Loop`` is for loop values, it's not the same than a ``switch``. So we need another keyword for that.